commands

```diff
@@ -196,15 +196,15 @@ Please check whether you have sufficient write privileges in $PWD"
 
       mongodb_username=$APP
 
-      # Have to set ENV vars this way. Using dokku config:set causes a loop where it tries to redeploy itself and calls this again
+      # We can use dokku config:set-norestart to avoid redeploy itself
 
-      set_dokku_env MONGODB_DATABASE "$mongodb_database"
-      set_dokku_env MONGODB_HOST "$mongodb_private_ip"
-      set_dokku_env MONGODB_PORT "$mongodb_internal_port"
-      set_dokku_env MONGODB_USERNAME "$mongodb_username"
-      set_dokku_env MONGODB_PASSWORD "$mongodb_password"
-      set_dokku_env MONGO_URL "mongodb://${mongodb_username}:${mongodb_password}@${mongodb_private_ip}:${mongodb_internal_port}/${mongodb_database}"
-      set_dokku_env MONGO_URI "mongodb://${mongodb_username}:${mongodb_password}@${mongodb_private_ip}:${mongodb_internal_port}/${mongodb_database}"
+      dokku config:set-norestart "$APP" MONGODB_DATABASE="$mongodb_database"
```

Comment:
I think these can all be done in one command now. Could you update the pull request to reflect that.